.github/workflows/build-and-upload-archives-on-schedule.yml

```diff
@@ -2,7 +2,7 @@ name: Venice Publication Pipeline
 
 on:
   schedule:
-    - cron: "30 20 * * 3"
+    - cron: "30 20 * * TUE,THU"
```

Comment:
Can you replace `TUE,THU` with `2,4`? Acc to Github docs, it supports [POSIX cron](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07) and I don't see the day names as supported in that spec